.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
 .secret
 proving_key.json
-verification_key.json
\ No newline at end of file
+verification_key.json
+build/
```

Comment:
How come `build` is gitignored, but I'm still seeing the JSON files inside `build` in this pull request?

Reply:
It was pushed previously to avoid us having to compile the circuits over and over, I believe.